tools/integration_tests/operations/operations_test.go

```diff
@@ -37,6 +37,12 @@ const SubDirInNonEmptyDestCopyDirectory = "subDestCopyDir"
 const DestCopyDirectoryNotExist = "notExist"
 const NumberOfObjectsInSrcCopyDirectory = 2
 const NumberOfObjectsInDestCopyDirectory = 2
+const DirectoryForListTest = "directoryForListTest"
+const FirstSubDirectoryForListTest = "firstSubDirectoryForListTest"
+const FileInDirectoryForListTest = "fileInDirectoryForListTest"
+const FileInFirstSubDirectoryForListTest = "fileInFirstSubDirectoryForListTest"
+const SecondSubDirectoryForListTest = "secondSubDirectoryForListTest"
+const FileInSecondSubDirectoryForListTest = "fileInSecondSubDirectoryForListTest"
```

Comment:
can we create third subdirectory which is empty ?

Reply:
added